e2e/test/scenarios/native-filters/reproductions/11480.cy.spec.js

```diff
@@ -32,11 +32,10 @@ describe("issue 11480", () => {
     SQLFilter.chooseType("Number");
     cy.location("search").should("eq", "?x=");
 
-    // When we run it again, the default has been cleared out so we get the right error.
-    SQLFilter.runQuery();
-    // eslint-disable-next-line no-unscoped-text-selectors -- deprecated usage
-    cy.contains(
-      "You'll need to pick a value for 'X' before this query can run.",
-    );
+    // We cannot run the query unless the default value for a required parameter is set.
+    cy.findByTestId("native-query-editor-container")
+      .icon("play")
+      .parents("button")
+      .should("be.disabled");
```

Comment:
@nemanjaglumac please have a look. In this PR, we're making default value mandatory for required parameters in native queries. This means, we can no longer attempt to run a query without a default value set — instead, I am checking if the run button is disabled.

Reply:
I'd maybe narrow this down to the native query editor sidebar.

Even better, we can try getting this element directly by its aria-label.